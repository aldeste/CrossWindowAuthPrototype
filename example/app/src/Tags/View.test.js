import View from "./View";
import renderer from "react-test-renderer";
import React from "react";
// BUG: Theres a bug with jest-styled-components and styled-components 2.0.0-17, fix when updated.
// import { matcher, serializer } from "jest-styled-components";
//
// expect.addSnapshotSerializer(serializer);
// expect.extend(matcher);
//
// Once fixed, replace toMatchSnapshot() with toMatchStyledComponentsSnapshot()

describe("View tag", () => {
  it("Should be defined", () => {
    expect(View).toBeDefined();
  });

  it("Is a styled component", () => {
    expect(View.name).toBe("StyledComponent");
  });

  it("Matches earlier configuration", () => {
    const tree = renderer.create(<View />).toJSON();
    tree.props.className = "";
    expect(tree).toMatchSnapshot();
  });
});
