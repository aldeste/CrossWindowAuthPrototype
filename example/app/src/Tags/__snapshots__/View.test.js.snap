// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`View tag Matches earlier configuration 1`] = `
<div
  className=""
/>
`;
